.github/workflows/pr.yml

```
@@ -453,6 +453,10 @@ jobs:
           - image: ghcr.io/ponylang/ponyc-ci-x86-64-unknown-linux-ubuntu24.04-builder:20240425
             debugger: lldb
             directives: runtimestats
+          - image: ghcr.io/ponylang/ponyc-ci-x86-64-unknown-linux-ubuntu24.04-builder:20240425
```

```suggestion
          - image: ghcr.io/ponylang/ponyc-ci-x86-64-unknown-linux-ubuntu24.04-builder:20250115
```

If that image works, and fixes the build issue, I'll push the change to the ponyc repo.